cmd/podman/common/create_opts.go

```diff
@@ -286,7 +286,7 @@ func ContainerCreateToContainerCLIOpts(cc handlers.CreateContainerConfig, rtc *c
 		LogDriver:         cc.HostConfig.LogConfig.Type,
 		LogOptions:        stringMaptoArray(cc.HostConfig.LogConfig.Config),
 		Name:              cc.Name,
-		OOMScoreAdj:       cc.HostConfig.OomScoreAdj,
```

Comment:
This looks suspicious - this is for the Docker-compat API, and I don't see any other place it could be set for said API, so we're basically just ignoring OOMScoreAdj for the Docker-compat create endpoint now?

Reply:
I guess it is fine in this case to override the default as we cannot say whether it is set to 0 or undefined.

I've reverted this line.